scripts/generate_frameID_json.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# Script generates a JSON file that contains all of the CANBUS frame IDs
+# defined within frame_parser.h (frame_parser.h is single source of truth)
+
+# Result JSON file will be stored as /network-table/src/frame_ids.json
+
+import json
+
+def read_SSOT():
+    id_array = []
+    frame_parser = open("../src/uccm-sensors/frame_parser.h", "r")
+    single_line = ""
+    
+    # Read until we see the // ID BEGIN
+    while "DON'T KNOW THE ACTUAL IDS YET" not in single_line:
```

Comment:
This code depends on some comment simon wrote. Can you make it less flaky, maybe search for lines that match a regex "^#define * 0x*" (i forget pythons regex syntax, basically match any line that starts with #define, followed by some string, followed by 0x* (a string starting with 0x) you get the idea.
We'll just have to assume any line matching this regex is an ID.

Reply:
can make the regex a little more specific:  "^#define * 0x*$"
I added the dollar sign at the end, which matches the end of the line